dotCMS/src/main/java/com/dotcms/cdi/CDIUtils.java

```diff
@@ -21,15 +21,29 @@ private CDIUtils() {
      * @return an Optional with the bean if found, empty otherwise
      */
     public static <T> Optional<T> getBean(Class<T> clazz) {
+            try {
+                return Optional.of(getBeanThrows(clazz));
+            } catch (Exception e) {
+                // Exception is already logged in getBeanThrows
+            }
+            return Optional.empty();
+    }
+
+
+    /**
+     * Get a bean from CDI container
```

Comment:
As getBean and getBeanThrows are both public, I would add a better javadoc to understand the difference between them

Reply:
done